postgres_backup.sql

```diff
@@ -0,0 +1,68 @@
+-- -------------------------------------------------------------
+-- TablePlus 6.1.2(568)
+--
+-- https://tableplus.com/
+--
+-- Database: postgres
+-- Generation Time: 2024-07-26 06:42:08.3820
+-- -------------------------------------------------------------
+
+
+-- This script only contains the table creation statements and does not fully represent the table in the database. Do not use it as a backup.
+
+-- Table Definition
+CREATE TABLE "public"."case" (
+    "docketNumber" varchar NOT NULL,
+    "trialLocation" varchar,
+    "trialDate" varchar,
+    "docketNumberSuffix" varchar,
+    PRIMARY KEY ("docketNumber")
+);
+
+-- This script only contains the table creation statements and does not fully represent the table in the database. Do not use it as a backup.
+
+-- Table Definition
+CREATE TABLE "public"."message" (
+    "attachments" jsonb,
+    "caseStatus" varchar NOT NULL,
+    "caseTitle" varchar NOT NULL,
+    "completedAt" varchar,
+    "completedBy" varchar,
+    "completedBySection" varchar,
+    "completedByUserId" varchar,
+    "completedMessage" varchar,
+    "createdAt" varchar NOT NULL,
+    "docketNumber" varchar NOT NULL,
+    "from" varchar NOT NULL,
+    "fromSection" varchar NOT NULL,
+    "fromUserId" varchar NOT NULL,
+    "isCompleted" bool NOT NULL,
+    "isRead" bool NOT NULL,
+    "isRepliedTo" bool NOT NULL,
+    "leadDocketNumber" varchar,
+    "message" varchar NOT NULL,
+    "messageId" varchar NOT NULL,
+    "parentMessageId" varchar NOT NULL,
+    "subject" varchar NOT NULL,
+    "to" varchar NOT NULL,
+    "toSection" varchar NOT NULL,
+    "toUserId" varchar NOT NULL,
+    PRIMARY KEY ("messageId")
+);
+
+
+
+-- Indices
+CREATE UNIQUE INDEX "PK_22bea6459f33e75269daee9bb49" ON public."case" USING btree ("docketNumber");
+
+
+-- Indices
+CREATE INDEX "IDX_cca9e3abeff4b0939aa8a25b3e" ON public.message USING btree ("completedBySection");
```

Comment:
The general recommendation I have seen is to default to btree, but hash might be worth considering for uuid fields (so long as we don't have duplicates!), where range queries don't make sense.